CMakeLists.txt

```diff
@@ -118,6 +118,11 @@ include(GNUInstallDirs)
 include(autocmake_safeguards)
 include(custom_color_messages)
 
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set(CMAKE_INSTALL_PREFIX "/usr/local/psi4" CACHE PATH "Install path" FORCE)
```

Comment:
Will make this a public discussion. I prefer encapsulated, but I'm not adamant.

The main business installs to ``{CMAKE_INSTALL_PATH}/lib/psi4/core.so`` which isn't good python installation protocol anyways (should install to detected python's site-packages for import psi4 to do any good automatically), so I think an encapsulated installation makes more sense for non-conda